js/index.js

```diff
@@ -40,3 +40,127 @@ const siteContent = {
 // Example: Update the img src for the logo
 let logo = document.getElementById("logo-img");
 logo.setAttribute('src', siteContent["nav"]["img-src"])
+//logo.setAttribute("src", "img/logo.png")
+logo.src = "img/logo.png"
+
+const navBar = document.querySelectorAll(".container header nav a")
+navBar[0].textContent = "Services"
+navBar[1].textContent = "Product"
+navBar[2].textContent = "Vision"
+navBar[3].textContent = "Features"
+navBar[4].textContent = "About"
+navBar[5].textContent = "Contact"
+const header = document.querySelector("header")
+
+header.style.backgroundColor = "green"
+const newNavItem1 = document.createElement("a")
+const newNavItem2 = document.createElement("a")
+newNavItem1.href = "#"
+newNavItem2.href = "#"
+newNavItem1.textContent = "Home Page"
+newNavItem2.textContent = "DMCA Contact"
```

Comment:
You could also do both of these simultaneously by updating the `innerHTML` and filling out a full a tag in HTML.